src/helpers/snapshot.ts

```diff
@@ -65,7 +65,8 @@ export async function getResults(space, proposal, votes) {
         strategies,
         space.network,
         voters,
-        parseInt(proposal.snapshot)
+        parseInt(proposal.snapshot),
+        import.meta.env.VITE_SCORES_URL + '/api/scores',
```

Comment:
I think the env var should be the full url, without /api/scores
```suggestion
        import.meta.env.VITE_SCORES_URL,
```